Sources/XCTest/Public/XCTestMain.swift

```diff
@@ -12,14 +12,20 @@
 //  for running tests and some infrastructure for running them.
 //
 
+#if os(macOS)
+    import SwiftFoundation
+#else
+    import Foundation
+#endif
+
 #if os(Linux) || os(FreeBSD)
     import Glibc
-    import Foundation
-#else
+#elseif os(macOS)
     import Darwin
-    import SwiftFoundation
 #endif
 
+#if os(macOS) || os(Linux) || os(FreeBSD)
```

Comment:
What's the motivation for introducing this conditional? Does it fail to build in on other platforms for some reason?